pkg/strategy/ewoDgtrd/strategy.go

```diff
@@ -1274,6 +1264,31 @@ func (s *Strategy) Run(ctx context.Context, orderExecutor bbgo.OrderExecutor, se
 
 		color.HiBlue("WinRate: %f", float64(totalTP)/float64(totalTP+totalSL))
 
+		maString := "vwema"
+		if s.UseSma {
+			maString = "sma"
+		}
+		if s.UseEma {
+			maString = "ema"
+		}
+
+		color.HiYellow("----- EWO Settings -------")
```

Comment:
ensure that color.X uses stderr, because optizmier use stdout to get the filename

Reply:
done